viewer/src/glsl/base/updateFragmentColor.glsl

```diff
@@ -2,16 +2,32 @@
 #pragma glslify: hsv2rgb = require('../color/hsv2rgb.glsl')
 #pragma glslify: packIntToColor = require('../color/packIntToColor.glsl')
 
+uniform sampler2D colorDataTexture;
```

Comment:
Not sure how well this scales now as we add more inputs, but until now we have specified all inputs as arguments to ensure that shader parameters are defined one place and not scattered around. Should we consider moving this outside the subroutine as well? Should we create a struct for colorInputs to avoid having to add more and more fields?

Reply:
Agreed. Splitting this into a new function for now, which also reduces the number of inputs to updateFragmentColor.